python/tests/expressions/test_visitors.py

```diff
@@ -1647,73 +1641,6 @@ def test_manifest_evaluator_or():
     assert _create_manifest_evaluator(expr).eval(manifest)
 
 
-def test_bound_predicate_invert():
```

Comment:
These tests don't make any sense anymore since the classes are abstract now and can't be initialized